doc/digraph.xml

```diff
@@ -252,6 +252,89 @@ gap> DigraphVertexLabel(gr, 2);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphEdgeLabels">
```

Comment:
This section of documentation should state that you can only read/set the edge labels of a digraph without multiple edges.